site/source/docs/api_reference/html5.h.rst

```diff
@@ -2077,6 +2077,9 @@ Functions
   :returns: On success, a strictly positive value that represents a handle to the created context. On failure, a negative number that can be cast to an |EMSCRIPTEN_RESULT| field to get the reason why the context creation failed.
   :rtype: |EMSCRIPTEN_WEBGL_CONTEXT_HANDLE|
 
+.. c:function:: EMSCRIPTEN_WEBGL_CONTEXT_HANDLE emscripten_webgl_create_context(const char *target, const EmscriptenWebGLContextAttributes *attributes)
+
+see ``emscripten_webgl_create_context_ext`` for description.
 
```

Comment:
I think it would be more natural to have the old function first, and the `_ext` after? Then the last one can also say how it differs from the former.

Reply:
done